packages/components/src/QualityBar/SplitQualityBar.component.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import { StackHorizontal } from '@talend/design-system';
+
+import {
+	EnrichedQualityType,
+	QualityBarPercentages,
+	QualityCommonProps,
+	QualityType,
+} from './QualityBar.types';
+import { QualityBarRatioBars } from './QualityBarRatioBars.component';
+
+type SplitQualityBarProps = QualityCommonProps & {
+	percentages: QualityBarPercentages;
+	disabled?: boolean;
+	onClick?: (e: React.MouseEvent<HTMLElement>, data: { type: EnrichedQualityType }) => void;
+	getDataFeature?: (type: string) => string;
+};
+
+export function SplitQualityBar({
+	empty,
+	getDataFeature,
+	invalid,
+	na,
+	onClick,
+	valid,
+	percentages,
+	disabled,
+}: SplitQualityBarProps) {
+	const totalValues = (empty || 0) + (invalid || 0) + (na || 0) + (valid || 0);
+	const usedValues = { empty, invalid, na, valid };
+	const fwd = { getDataFeature, onClick };
+
+	if (disabled) {
+		return null;
```

Comment:
imo we should return the same kind of qb as we do for the others
i think that hiding it is more a project concern

Reply:
allright, i can make it this way 👍🏼